portfolio/src/main/webapp/script.js

```diff
@@ -94,6 +94,7 @@ function deleteComment(comment) {
   fetch('/delete-data', {method: 'POST', body: params});
 }
 
+//adds map to tea.html page
```

Comment:
nit: add space and capitalize first letter. Also add period. Comments should be sentences.

Reply:
ok thank you! I'll make these changes now and push them before merging into master.